src/source/PeerConnection/Rtp.h

```diff
@@ -58,7 +74,7 @@ STATUS freeKvsRtpTransceiver(PKvsRtpTransceiver*);
 
 STATUS kvsRtpTransceiverSetJitterBuffer(PKvsRtpTransceiver, PJitterBuffer);
 
-UINT64 convertTimestampToRTP(UINT64, UINT64);
+#define convertTimestampToRTP(clockRate, pts) (pts * clockRate / HUNDREDS_OF_NANOS_IN_A_SECOND)
```

Comment:
Macros should be capitalized and underscored - something like CONVERT_TIMESTAMP_TO_RTP

Reply:
Also, it's a good practice to wrap the params in parenthesis 

#define CONVERT_TIMESTAMP_TO_RTP(clockRate, pts) ((pts) * (clockRate) / HUNDREDS_OF_NANOS_IN_A_SECOND)

Reply:
ok